examples/instance/terraform/plugin.go

```diff
@@ -382,8 +382,8 @@ func (p *plugin) Provision(spec instance.Spec) (*instance.ID, error) {
 	// set the tags.
 	// add a name
 	if spec.Tags != nil {
-		if _, has := spec.Tags["Name"]; !has {
-			spec.Tags["Name"] = string(id)
+		if _, has := spec.Tags["name"]; !has {
+			spec.Tags["name"] = string(id)
```

Comment:
On AWS (at least) the `Name` tag has a particular meaning, that's the first field displayed in the default header of the EC2 console page. Changing the case of the tag will result in an empty value in the Name column. So this change will have a huge impact on AWS deployments.